docs/tiling.md

```diff
@@ -0,0 +1,337 @@
+# Integration of TileJSON with MicroJSON
+## Purpose
+This specification outlines how to use TileJSON to integrate tiled MicroJSON data. It provides examples of how TileJSON can be used to specify the tiling scheme and zoom levels for MicroJSON data. It is based on the [TileJSON 3.0.0 specification](https://github.com/mapbox/tilejson-spec/blob/master/3.0.0/README.md), but extends it by recommending additional properties include the integration of MicroJSON data and fit purposes of microscopy data visualization. Recommendations made here are NOT part of the original TileJSON specification, but are made to fit the purposes of microscopy data visualization and integration with MicroJSON, although all recommendations are made to be compatible with the original TileJSON specification.
```

Comment:
This can be rewritten as 
The recommendations provided here are not intrinsic to the original TileJSON specification but have been tailored to suit the needs of microscopy data visualization and integration with MicroJSON. However, all suggestions are designed to maintain compatibility with the original TileJSON specification

Reply:
Deleted all hidden python files

Reply:
Deleted files and changed as suggested